src/exodus/walletmodels.h

```diff
@@ -7,6 +7,7 @@
 
 #include "../serialize.h"
 #include "../uint256.h"
+#include "../pubkey.h"
```

Comment:
We should ordered by ascending to prevent duplicated include. So it should go before `serialize.h`.